kubernetes/server/create.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
 function kubectl_create() {
-  kubectl create -f scdf.yml
+  kubectl create -f scdf.yml --namespace $KUBERNETES_NAMESPACE
   READY_FOR_TESTS=1
   for i in $( seq 1 "${RETRIES}" ); do
-    SERVER_URI=$(kubectl get svc scdf | grep scdf | awk '{print $3}')
+    SERVER_URI=$(kubectl get svc scdf --namespace $KUBERNETES_NAMESPACE | grep scdf | awk '{print $3}')
     [ '<pending>' != $SERVER_URI ] && READY_FOR_TESTS=0 && break
```

Comment:
Hi @ilayaperumalg, it's weird because the netcat port is used to check local ports with nc, if you remove that then it will probably fail for local tests. I did not know that k8s was using it as well, but I think that your change may break local, have you tried local too?